src/index.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+require('./../style/main.scss');
+
+import 'whatwg-fetch';
+import $ from 'jquery';
+import urlJoin from 'url-join';
+import './errors';
+import { parseMovies, renderMovie } from './Movies';
+import { fetchDocument, parseYear } from './Utils';
+import { proxyUrl, baseUrl } from './config';
+
+const ENTER = 13;
+
+// flash errors
+const messages = {
+  [Error.INVALID_INPUT]: 'Please enter a valid year',
+  [Error.NO_INPUT]: 'Please enter a valid year',
+  [Error.OUT_OF_RANGE]: 'Year is out of range ( 1950 - 2030 )',
+  [Error.NO_RESULTS]: 'Oh Snap! No Queries Matched... Please try again!'
+};
+
+class Loader {
+  constructor(loader){
+    this.$loader = $(loader);
+  }
+  start() {this.$loader.fadeIn('fast')};
+  stop () {this.$loader.fadeOut('fast')};
```

Comment:
If you really want to save vertical space:
```js
class Loader {
  constructor(el) { this.$el = $(el); }
  start() { this.$el.fadeIn('fast'); }
  stop () { this.$el.fadeOut('fast'); }
}
```

Note:
- missing semicolons
- missing spaces around _curlies_
- redundant semicolons after method definition